### DISCUSSION TITLE
Custom filtering rules ttl

### DISCUSSION THREAD

**Discussion by Isrofilov** [AdguardTeam/AdGuardHome#3155] [Q&A]
if i allow the client a specific domain. it receives an cached response 0.0.0.0
https://imgur.com/a/Jb4nWmC

v0.106.2

**Comment by ameshkov** [comment]
TTL is controlled by `blocked_response_ttl` setting in `AdGuardHome.yaml` (see https://github.com/AdguardTeam/AdGuardHome/wiki/Configuration)

**Reply by ameshkov** [reply]
Ah, and check what upstream is used. Maybe you're using an upstream with filtering of its own?
